deploy_and_run_tests.sh

```diff
@@ -225,14 +225,18 @@ python gcp_variant_transforms/testing/integration/run_vcf_to_bq_tests.py \
     --staging_location "gs://${gs_dir}/staging" \
     --temp_location "gs://${gs_dir}/temp" \
     --logging_location "gs://${gs_dir}/temp/logs" \
-    --image "${full_image_name}" ${TEST_ARGUMENTS}
-
+    --image "${full_image_name}" ${TEST_ARGUMENTS} &
+pid_vcf_to_bq=$!
 if [[ -n "${run_preprocessor_tests}" ]]; then
   python gcp_variant_transforms/testing/integration/run_preprocessor_tests.py \
       --project "${project}" \
       --staging_location "gs://${gs_dir}/staging" \
       --temp_location "gs://${gs_dir}/temp" \
       --logging_location "gs://${gs_dir}/temp/logs" \
-      --image "${full_image_name}"
+      --image "${full_image_name}" &
+fi
+pid_preprocess=$!
+if wait $pid_vcf_to_bq && wait $pid_preprocess; then
+  color_print "$0 succeeded!" "${GREEN}"
+else exit 1
```

Comment:
nit: consider moving `exit 1` to next line (just for consistency with the rest of the file).

Reply:
Right, Thanks!